snuba/cli/multistorage_consumer.py

```diff
@@ -113,8 +119,10 @@
 )
 def multistorage_consumer(
     storage_names: Sequence[str],
-    consumer_group: str,
+    raw_events_topic: Optional[str],
     commit_log_topic: str,
+    consumer_group: str,
+    bootstrap_server: Sequence[str],
```

Comment:
Shouldn't this be optional? Most of the times we wouldn't want consumers to provide the bootstrap_server.

Reply:
Since it's a sequence, if it's not provided it's just an empty list and the default value gets applied in that case. This is the same way it's done in the other consumer CLI.